### PR TITLE
added guardian-data endpoint swagger doc

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Added swagger docs for the guardian-data endpoint
image

### CODE REVIEW

**Review by AdoAdoAdo — APPROVED** [review]

**Review by schimih — DISMISSED** [review]

File: cmd/proxy/config/swagger/openapi.json [hunk 1]
```diff
@@ -2985,6 +3020,51 @@
...
+        }
+      },
+      "GuardianResponse": {
+        "type": "object",
+        "properties": {
+          "data": {
+            "type": "object",
+            "properties": {
+              "activeGuardian": {
+                "type": "array",
```
Comment (sstanculeanu) [inline cmd/proxy/config/swagger/openapi.json:3044]:
```suggestion
                "type": "object",
```

File: cmd/proxy/config/swagger/openapi.json [hunk 2]
```diff
@@ -2985,6 +3020,51 @@
...
+                "items": {
+                  "$ref": "#/components/schemas/Guardian"
+                }
+              },
+              "pendingGuardian": {
+                "type": "array",
```
Comment (sstanculeanu) [inline cmd/proxy/config/swagger/openapi.json:3050]:
```suggestion
                "type": "object",
```

File: cmd/proxy/config/swagger/openapi.json (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/proxy/config/swagger/openapi.json:3044]: it won't work (it won't display the fields inside the object)

File: cmd/proxy/config/swagger/openapi.json (hunk 2 above)
Comment (bogdan-rosianu) [reply cmd/proxy/config/swagger/openapi.json:3050]: it won't work (it won't display the fields inside the object)

**Review by sstanculeanu — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**